### PR TITLE
Milestone 2: implementing components

### PR DESCRIPTION
**Pull request by sediqullahbadakhsh**
in this milestone using react class-based components implemented Calculator buttons and rendered inside App.js

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```
Comment (micheaol) [inline src/App.js]: Please ensure that the `App.js` component is class-based according to the project requirement.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  }
+
+  render() {
+    return (
+      <div className="calculator">
+        <div className="row">
+          <input className="input" value="0" />
+        </div>
+        <div className="row">
+          <p className="colr-grey item">AC</p>
```
Comment (micheaol) [inline src/components/Calculator.js:17]: Please ensure that the `Calculator.js` component should match the following given design.

Your Implementation | Figma Design
:---:|:---:
image | image

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @sediqullahbadakhsh , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/App.js (hunk 1 above)
Comment (sediqullahbadakhsh) [reply src/App.js]: Thanks solved

File: src/components/Calculator.js (hunk 2 above)
Comment (sediqullahbadakhsh) [reply src/components/Calculator.js:17]: Thanks solved

**Review by nicupop729 — APPROVED** [review]
<h1>Status: Approved 🥇 ✔</h1>
Hi @sediqullahbadakhsh,

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

### Some highlights for you
* Correct gitflow practice ✔️ 
* Good commit messages ✔️ 
* Professional README ✔️ 
* Linters are passing :heavy_check_mark: 
* Clean codebase ✔️ 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**